.travis.yml

```diff
@@ -4,9 +4,9 @@ python:
   - "3.3"
   - "3.4"
 env:
-  - DJANGO='django>=1.4,<1.5'
   - DJANGO='django>=1.6,<1.7'
   - DJANGO='django>=1.7,<1.8'
+  - DJANGO='--pre django'
```

Comment:
Probably worth allowing this build to fail with [`allow_failures`](http://docs.travis-ci.com/user/build-configuration/#Rows-That-are-Allowed-To-Fail).

It's good that we're checking the newer version, but I don't think that it should be blocking.